bundles/org.smarthomej.binding.amazonechocontrol/src/main/java/org/smarthomej/binding/amazonechocontrol/internal/handler/AccountHandler.java

```diff
@@ -141,6 +144,8 @@ public class AccountHandler extends BaseBridgeHandler implements PushConnection.
     private boolean disposing = false;
     private @Nullable AccountTO accountInformation;
 
+    Set<String> deviceSerialNumbers = Collections.synchronizedSet(new LinkedHashSet<>());
```

Comment:
Why don't you use `ConcurrentHashMap.newKeySet()`? That's what we usually do if we want to create a new thread-safe `Set`.